translator/trace/protospan_to_jaegerthrift.go

```diff
@@ -37,6 +37,10 @@ var (
 	errWrongLenID      = errors.New("ID does not have 8 bytes")
 )
 
+var (
+	unknownProcess = &jaeger.Process{ServiceName: "unknown-service-name"}
```

Comment:
Could we perhaps just skip "unknown-service-name" and instead make it
```go
var unsetNonNilProcess = new(jaeger.Process)
```

```go
if node == nil {
   // Jaeger requires a non-nil process
   // regardless of if node is nil.
    // See Issue https://github.com/census-instrumentation/opencensus-service/issues/332
   return unsetNonNilProcess
}
```

Reply:
"unknown-service-name" is what is used by Jaeger itself in such cases see https://github.com/jaegertracing/jaeger/blob/774c01b156f81d14ca5a1e99e1f7b6453e406d00/model/converter/thrift/zipkin/to_domain.go#L32-L33